src/cert.rs

```diff
@@ -173,6 +175,17 @@ impl<'a> Cert<'a> {
         self.subject.as_slice_less_safe()
     }
 
+    /// Get the RFC 5280-compliant [`SubjectPublicKeyInfo`] (SPKI) of this [`Cert`].
```

Comment:
This stale ref to the old name is causing a rust docs error in CI:

```suggestion
    /// Get the RFC 5280-compliant [`SubjectPublicKeyInfoDer`] (SPKI) of this [`Cert`].
```